stories/card.stories.js

```diff
@@ -1,88 +1,158 @@
-            <Card theme={dark} depth={2}>
-              <p>Depth = 2</p>
-            </Card>
+      <Card theme={theme}>
+        <h1>Cards</h1>
+        <h3>Interface</h3>
+        <SyntaxHighlighter language="reason" style={okaidia} showLineNumbers>
+          {`type Card: (
+  ~spacing: option(CardStyles.spacing),
+  ~theme: option(UiTypes.theme),
+  ~depth: option(int),
+  ~header: option(React.Element),
+  ~footer: option(React.Element),
+  ~children: option(React.Element),
+  ~className: option(string),
+  ~styles: option(array(Css.rule), /* DEPRECATED -- These will be removed in future release in favor of "className" */
+) => React.element;`}
+        </SyntaxHighlighter>
+
+        <h3>Preview</h3>
+        <h4>Padding</h4>
+        <div style={smallCardContainer}>
+          <Card spacing={tiny} depth={1} theme={theme}>
+            <h2>Tiny Padding</h2>
+          </Card>
+        </div>
+        <div style={smallCardContainer}>
+          <Card depth={1} theme={theme}>
+            <h2>Normal Padding</h2>
+          </Card>
+        </div>
+        <div style={smallCardContainer}>
+          <Card spacing={large} depth={1} theme={theme}>
+            <h2>Large Padding</h2>
           </Card>
-        </Card>
-      </div>
-      <div style={nestedCardContainer}>
-        <Card theme={light}>
-          <p>Depth = 0</p>
-          <Card theme={light} depth={1}>
-            <p>Depth = 1</p>
-            <Card theme={light} depth={2}>
-              <p>Depth = 2</p>
+        </div>
+        <div style={smallCardContainer}>
+          <Card spacing={huge} depth={1} theme={theme}>
+            <h2>Huge Padding</h2>
+          </Card>
+        </div>
+
+        <h4>Nesting</h4>
+        <p>
+          Using the 'Depth' Parameter, one can stack multiple cards. The
+          stacking here actuall starts with the background, which is one of our
+          cards.
+        </p>
+        <div style={nestedCardContainer}>
+          <Card spacing={tiny} depth={1} theme={theme}>
+            <h2>No. 1</h2>
+            <Card spacing={tiny} depth={2} theme={theme}>
+              <h2>No. 2</h2>
+              <Card spacing={tiny} depth={3} theme={theme}>
+                <h2>Cards all the way down</h2>
+              </Card>
             </Card>
           </Card>
-        </Card>
-      </div>
+        </div>
+
+        <h4>Header / Footer</h4>
+        <p>
+          One can pass in a header and footer component. To be as flexible as
+          possible here, we do not implement any spacing options here. The
+          component passed into the header should place itself according to the
+          space available
+        </p>
+        <div style={nestedCardContainer}>
+          <Card
+            header={
+              <div style={headerStyles}>
+                <h3>Some Header</h3>
+              </div>
+            }
+            footer={
+              <div style={headerStyles}>
+                <h3>Some Header</h3>
```

Comment:
```suggestion
                <h3>Some Footer</h3>
```

Reply:
I've updated it and added your suggestion 👍